test/functionalTest/cases/690_geoquery_crash/two_geoscopes.test

```diff
@@ -0,0 +1,112 @@
+# Copyright 2014 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Simple NGSI10 subscription for EXACT city and isPattern and deeper service path
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 0-255
+
+--SHELL--
+
+#
+# 01. Make the issue690 query with two geoscopes and make sure the broker does not crash
+#
+
+echo "01. Make the issue690 query with two geoscopes and make sure the broker does not crash"
+echo "======================================================================================"
+payload='{
+  "entities":
+  [
+    {
+      "type": "",
+      "isPattern": "true",
+      "id": ".*"
+    }
+  ],
+  "restriction":
+  {
+    "scopes": [
+      {
+        "type" : "FIWARE::Location",
+        "value" :
+        {
+          "polygon":
+          {
+            "vertices": [
+              { "latitude": "-33.603458", "longitude": "-70.652788" },
+              { "latitude": "-33.604231", "longitude": "-70.646032" },
+              { "latitude": "-33.606123", "longitude": "-70.646781" },
+              { "latitude": "-33.606123", "longitude": "-70.646781" }
+            ]
+          }
+        }
+      },
+      {
+        "type" : "FIWARE::Location",
```

Comment:
At the end, this second scope with a circle is not relevant for the bug and associated fix. Thus, I'd suggest to remove it.